### PR TITLE
[semver:patch] Add Repository Strucutre - 0.0.1

### PR DESCRIPTION
**Pull request by Cwagne17**
Adds the structure of the skeleton for the orb. Setting up for development process

### CODE REVIEW

File: src/README.md [hunk 1]
```diff
@@ -1,26 +1,41 @@
...
-    Specify version 2.1 for orb-compatible configuration `version: 2.1`
-2. **description**
-    Give your orb a description. Shown within the CLI and orb registry
-3. **display**
-    Specify the `home_url` referencing documentation or product URL, and `source_url` linking to the orb's source repository.
-4. **orbs**
-    (optional) Some orbs may depend on other orbs. Import them here.
+### How to Contribute
+
+We welcome [issues](https://github.com/<organization>/<project-name>/issues) to and [pull requests](https://github.com/<organization>/<project-name>/pulls) against this repository!
```
Comment (StarChaser44) [inline src/README.md:20]:
```suggestion
We welcome [issues](https://github.com/<organization>/<project-name>/issues) and [pull requests](https://github.com/<organization>/<project-name>/pulls) against this repository!
```

File: src/jobs/lint-app.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+description: >
+  This job will lint the application based on what language the application is written in.
```
Comment (StarChaser44) [inline src/jobs/lint-app.yml:2]: I would love to have a meeting to understand how this will be implemented for multiple languages.

**Review by StarChaser44 — APPROVED** [review]
Looks good. If this is still a WIP re-request me!

File: src/jobs/lint-app.yml (hunk 2 above)
Comment (Cwagne17) [reply src/jobs/lint-app.yml:2]: Sounds good

File: .circleci/test-deploy.yml [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+      - checkout
+      - orb/check-docker-semver
+      - orb/setup-aws
+      - orb/setup-docker
+      - orb/tag-repo
+      - orb/validate-num-params
+      - orb/validate-str-params
+
+workflows:
+  test-deploy:
```
Comment (EVogel1999) [inline .circleci/test-deploy.yml:21]: Will test deploy actually deploy to eb/ecs/etc?

**Review by EVogel1999 — COMMENTED** [review]
Just have one quick question.

File: .circleci/test-deploy.yml (hunk 3 above)
Comment (Cwagne17) [reply .circleci/test-deploy.yml:21]: No

**Pull request merged**